plugins/tracers/meson.build

```diff
@@ -0,0 +1,56 @@
+# Library source files
+libgst_shark_src_files = [
+    'gstdot.c',
+	'gstcpuusagecompute.c',
+	'gstproctimecompute.c',
+	'gstctf.c',
+	'gstparser.c'
+]
+
+# libgst_shark_header_files = []
+
+libgst_shark_c_args = [gst_c_args,
+                      '-DGST_USE_UNSTABLE_API',
+                      '-Wno-undef']
```

Comment:
Could you describe why we should use unstable API?

Reply:
I just used it from here: https://github.com/RidgeRun/gst-shark/blob/master/plugins/tracers/Makefile.am#L15

But I don't see any place on the code that actually uses it, would you suggest removing it?

Reply:
Hmmm... Ok. I think we can remove it if we are not using unstable API. Otherwise, we should explicitly specify why it's needed